app/views/extended_metadata_types/_help_info.erb

```diff
@@ -0,0 +1,8 @@
+<!-- the links will be updated after the official doc is available -->
+<div>
+  For more information, please visit our
+  <%= link_to 'Extended Metadata Technical Overview', 'https://github.com/seek4science/seek-documentation/blob/gh-pages-extended_metadata-type/tech/extended_metadata/extended-metadata-type.md', target: '_blank' %> page.
+  This is <%= link_to 'JSON schema', 'https://github.com/seek4science/seek-documentation/blob/gh-pages-extended_metadata-type/tech/extended_metadata/extended_metadata_type_schema.json', target: '_blank' %> used to define your JSON file for creating a new extended metadata type via upload.
+  Here’s a <%= link_to 'step-by-step example', 'https://github.com/seek4science/seek-documentation/blob/gh-pages-extended_metadata-type/tech/extended_metadata/a-complete-example.md', target: '_blank' %> of how to create a new extended metadata type.
```

Comment:
use the help_dictionary for these links, where they can then also be updated when they go live